src/Microsoft.NET.Sdk.Functions.Generator/Program.cs

```diff
@@ -52,5 +43,20 @@ private static void Main(string[] args)
                 }
             }
         }
+
+        internal static void ParseArgs(string[] args, out string assemblyPath, out string outputPath, out bool functionsInDependencies, out IEnumerable<string> excludedFunctionNames)
+        {
+            assemblyPath = args[0].Trim();
+            outputPath = args[1].Trim();
+            functionsInDependencies = bool.Parse(args[2].Trim());
+
+            excludedFunctionNames = Enumerable.Empty<string>();
+
+            if (args.Length > 3)
+            {
+                var excludedFunctionNamesArg = args[3].Trim();
```

Comment:
If you look above -- we were using `args[2]` previously -- which is meant for `functionsInDependencies`.